terraform/ide.tf

```diff
@@ -38,6 +38,11 @@ VPC_CIDR=${try(module.cluster.vpc_cidr, "")}
 VPC_PRIVATE_SUBNET_ID_0=${try(module.cluster.private_subnet_ids[0], "")}
 VPC_PRIVATE_SUBNET_ID_1=${try(module.cluster.private_subnet_ids[1], "")}
 VPC_PRIVATE_SUBNET_ID_2=${try(module.cluster.private_subnet_ids[2], "")}
+AIML_DL_IMAGE=763104351884.dkr.ecr.${data.aws_region.current.name}.amazonaws.com/pytorch-inference-neuron:1.12.1-neuron-py38-sdk2.5.0-ubuntu20.04
+AIML_NEURON_BUCKET_NAME=${try(module.cluster.aiml_neuron_s3_bucket_name, "")}
+AIML_NEURON_ROLE_ARN=${try(module.cluster.aiml_neuron_role_arn, "")}
+AI_ML_SUBNET=${module.cluster.private_subnet_ids[0]},${module.cluster.private_subnet_ids[1]},${module.cluster.private_subnet_ids[2]}
```

Comment:
How would you like to change it to "AIML_SUBMET" for consistency with other variables "AIML_NEURON_XXX"?